client.go

```diff
@@ -101,7 +136,7 @@ func (c *StatsdClient) CreateSocket() error {
 		return err
 	}
 	c.conn = conn
-	c.conn_type = "udp"
+	c.connType = "udp"
```

Comment:
If i am reading this correctly, irrespective of the c.connType , you are setting it to `udp` .Any specific reason ?

Reply:
There is a separate function `CreateTCPSocket()` for creating TCP sockets.

Reply:
The type of connection is being store upon creation so that the reconnect function knows which create socket function to call. I think this is a better answer to your question.